tests/bindings-xcframework-test/bindings-xcframework-test.csproj

```diff
@@ -17,6 +17,8 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\product.snk</AssemblyOriginatorKeyFile>
     <TargetFrameworkIdentifier>Xamarin.iOS</TargetFrameworkIdentifier>
+    <MtouchArch Condition="'$(Platform)' == 'iPhoneSimulator'">x86_64</MtouchArch>
+    <MtouchArch Condition="'$(Platform)' == 'iPhone'">arm64</MtouchArch>
```

Comment:
Binding projects shouldn't need to know an MtouchArch value, they should package the entire .xcframework, not resolve to a specific variation of it.

Reply:
It is what's being packaged

```
$ l tests/xharness/tmp-test-dir/xcframework-test648/bin/Any\ CPU/Debug-tvos/bindings-xcframework-test.resources/XTest.xcframework/
total 8
-rw-r--r--  1 poupou  staff  2794 30 Nov 15:27 Info.plist
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 ios-arm64_armv7_armv7s
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 ios-i386_x86_64-simulator
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 macos-x86_64
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 tvos-arm64
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 tvos-x86_64-simulator
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 watchos-arm64_32_armv7k
drwxr-xr-x  3 poupou  staff    96 30 Nov 15:27 watchos-i386_x86_64-simulator
```

but it still try to resolve it when it's not actually needed (for bindings) and that does not show up in other platform since they default exists in `Xamarin.shared.targets`.

I'll look at skipping the resolve step for bindings